test/unittest/common/cavl/test_cavl.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <libcyphal/common/cavl/cavl.hpp>
 
+#include <gmock/gmock.h>
```

Comment:
This file can be skipped from review b/c it was reviewed at the original pavel-kirienko/cavl repo.